packages/shared/src/lib/query.ts

```diff
@@ -147,6 +147,7 @@ export enum RequestKey {
   Ads = 'ads',
   FeedByIds = 'feedByIds',
   SlackChannels = 'slack_channels',
+  UserIntegration = 'user_integration',
   UserIntegrations = 'user_integrations',
```

Comment:
I think it can actually just reuse `user_integrations` if we add `id` as I noted in previous comment.

Reply:
Hmm yeah let's do that, so we can invalidate on 1 top level :)